modules/refine-local-runner/pom.xml

```diff
@@ -98,6 +98,11 @@
       <artifactId>vavr</artifactId>
       <version>0.10.4</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
+      <version>${zstd-jni.version}</version>
+    </dependency>
```

Comment:
This pulls in this new dependency (BSD-licensed) to support the Zstd compression format.